packages/docs/package.json

```diff
@@ -15,7 +16,7 @@
     "@fuel-ui/icons": "0.23.3",
     "@fuel-ui/react": "0.23.3",
     "@fuels/connectors": "0.4.0",
-    "@fuels/react": "0.20.0",
+    "@fuels/react": "0.21.0-main-30f37c8",
```

Comment:
can we use `0.21.0` directly? it's being released already

Reply:
yes, for sure. I'll update it here

Reply:
https://github.com/FuelLabs/fuels-wallet/pull/1346/commits/a20145a09fc917e48a831027188d572170687ce7